examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -322,18 +320,23 @@
     "    \n",
     "    dt = model0.critical_dt\n",
     "\n",
-    "    geometry, rec = load_shot_data(param['shot_id'], dt)\n",
-    "    geometry.model = model0\n",
-    "    # Set up solver.\n",
-    "    solver = AcousticWaveSolver(model0, geometry, space_order=4)\n",
+    "    # Get src_position and data\n",
+    "    src_positions, rec = load_shot_data(param['shot_id'], dt)\n",
+    "\n",
+    "    # Set attributes to solver\n",
+    "    solver.model=model0\n",
```

Comment:
changing state from the outside world is bad and could have unpredictable side effects

one should instead `rebuild`

Reply:
`Model` has a `vp.setter` wrapper (that we talked about before) so we should to able to use that.